src/helpers/overrides.js

```diff
@@ -75,6 +75,21 @@ export function toObjectOverride<T>(override: OverrideT<T>): OverrideT<T> {
   return override;
 }
 
+/**
+ * Get a convenient override object that will always have {component, props}
+ */
+export function getOverrideObject<T>(
```

Comment:
I added this because it seems like in most cases you'll need to pass-thru the `props` and `style` anyway. Thoughts here? @schnerd

Reply:
I suppose this does feel a bit cleaner. I'm a fan.

Reply:
Great, I'll add tests